versions/1/data_stream/fields/fields.spec.yml

```
@@ -6,66 +6,82 @@ spec:
   type: array
   items:
     type: object
+    # TODO: We should disable additionalProperties. There are properties used in
+    # elastic/integrations that have no purpose or are typos.
     additionalProperties: true
```

+1, specially after adding these missing properties. @mtojek wdyt?
```suggestion
    # TODO: We should disable additionalProperties. There are properties used in
    # elastic/integrations that have no purpose or are typos.
    additionalProperties: false
```

---

+1 on my side too.

I think that we left it open to allow for extra properties like examples, patterns, etc. It seems that we aren't using the actively, so let's close this option.

---

I set `additionalProperties: false` and adjusted some of the test packages to remove level, group, and title.

---

There are many fields using level, group and/or title in the integrations repo, wdyt about adding these fields here by now? We could add them directly as deprecated, and eventually phase them out.

I have created this PR to check what would fail in the integrations repo after this change: https://github.com/elastic/integrations/pull/3024